docs/docs/releases/0.2.0.md

```diff
@@ -1,302 +1,550 @@
+[#4032](https://github.com/mathesar-foundation/mathesar/pull/4032 "Remove fixture util")
+[#4039](https://github.com/mathesar-foundation/mathesar/pull/4039 "Added favicon icons for the page templates in base.html and added respective icons in the static folder")
+[#4045](https://github.com/mathesar-foundation/mathesar/pull/4045 "Add spell-checking to MkDocs build process")
+[#4060](https://github.com/mathesar-foundation/mathesar/pull/4060 "Bump django from 4.2.16 to 4.2.17")
+[#4064](https://github.com/mathesar-foundation/mathesar/pull/4064 "Rename references to mathesar/mathesar-ansible to mathesar-infrastructure")
+[#4076](https://github.com/mathesar-foundation/mathesar/pull/4076 "Remove TODO_BETA code comments")
+[#4125](https://github.com/mathesar-foundation/mathesar/pull/4125 "Update codeql-analysis.yml to use latest codeql dependencies")
+[#4133](https://github.com/mathesar-foundation/mathesar/pull/4133 "Update versions and remove unused dependency")
+[#4145](https://github.com/mathesar-foundation/mathesar/pull/4145 "Reset django migrations, document upgrade")
+[#3980](https://github.com/mathesar-foundation/mathesar/pull/3980 "Fix row height calculation during grouping removal")
+[#4158](https://github.com/mathesar-foundation/mathesar/pull/4158 "Updates for file mathesar\_ui/src/i18n/languages/en/dict.json in ja")
+[#4159](https://github.com/mathesar-foundation/mathesar/pull/4159 "Updates for file translations/en/LC\_MESSAGES/django.po in ja")
+[#4165](https://github.com/mathesar-foundation/mathesar/pull/4165 "Identify i18n translate function calls made with variables instead of strings")
+[#4152](https://github.com/mathesar-foundation/mathesar/pull/4152 "Release 0.2.0")
+[#4170](https://github.com/mathesar-foundation/mathesar/pull/4170 "Rpc performance")
 
 ## Upgrading to 0.2.0 {:#upgrading}
 
-We do not fully support upgrading from previous versions to `0.2.0`. In particular, you will lose any Explorations and User accounts (e.g., usernames and passwords) you created in Mathesar during the process. Furthermore, manual steps are required.
+Upgrading to version 0.2.0 will:
 
-### Prerequisite steps (run before starting Mathesar version 0.2.0)
+- **Remove** out your [user](../user-guide/users.md) accounts, passwords, [explorations](../user-guide/data-explorer.md), and [metadata](../user-guide/metadata.md). You will need to recreate these after the upgrade.
+- **Keep** your databases and the data within them.
 
-If you want to reuse the same database server for Mathesar's internal database, you need to drop the `mathesar_django` database and recreate it on that server, using the same PostgreSQL role that you initially used to install Mathesar, or a role with superuser privileges.
+!!! question "Should I upgrade or reinstall?"
+    - If any of your connected databases reside inside Mathesar's internal database server, then you will need to perform the upgrade steps below or export your data (e.g. via `pg_dump`) and reinstall Mathesar.
```

Comment:
@mathemancer @seancolsen 

Okay, on further reading, this misrepresents the upgrade process to the user.

We keep the *underlying* database, but the databases configured in Mathesar still get removed. **The user would still have to configure them back manually**.

This isn't mentioned anywhere.

Reply:
I've raised https://github.com/mathesar-foundation/mathesar/pull/4191 which should fix this.